docs/clients/promtail/pipelines.md

```diff
@@ -153,30 +153,37 @@ scrape_configs:
 The following sections further describe the types that are accessible to each
 stage (although not all may be used):
 
-##### Label Set
+#### Label Set
 
 The current set of labels for the log line. Initialized to be the set of labels
 that were scraped along with the log line. The label set is only modified by an
 action stage, but filtering stages read from it.
 
 The final label set will be index by Loki and can be used for queries.
 
-##### Extracted Map
+#### Extracted Map
 
 A collection of key-value pairs extracted during a parsing stage. Subsequent
 stages operate on the extracted map, either transforming them or taking action
 with them. At the end of a pipeline, the extracted map is discarded; for a
 parsing stage to be useful, it must always be paired with at least one action
 stage.
 
-##### Log Timestamp
+The extracted map is initialized with the same set of initial labels that were
+scraped along with the log line. This initial data allows pipeline stages - which
+can only manipulate the extracted map - to take actions on the values of initial
+labels too. For example, log entries tailed from files have the initial label
+`filename` whose value is the file path, and the initial extracted map contains
+the same `filename` key-value pair as well.
```

Comment:
I'm a little confused by the sentence that explains the use case:

> This initial data allows pipeline stages - which can only manipulate the extracted map - to take actions on the values of initial labels too.

It seems to imply there are stages that aren't able to do anything except manipulate the extracted map. Does this sound any better? 

> This initial data allows for taking action on the values of labels inside pipeline stages that only manipulate the extracted map.

I'd like to change the phrasing on this just a little bit: 

> For example, log entries tailed from files have the initial label `filename` whose value is the file path, and the initial extracted map contains the same `filename` key-value pair as well.

To something like this, breaking up the sentences:

> For example, log entries tailed from files have the label `filename`, whose value is the file path that was tailed. When a pipeline executes for that log entry, the initial extracted map would contain also `filename` using the same value as the label.   

WDYT?

Reply:
Your version is definitely better. Thanks for helping me rephrasing it in a clearer way 🙏